jekyll/_posts/2021-09-25-version-160-released.md

```diff
@@ -0,0 +1,817 @@
+  cycle collection process. See the documentation of `=trace` for more details.
+- On embedded devices `malloc` can now be used instead of `mmap` via `-d:nimAllocPagesViaMalloc`.
+  This is only supported for `--gc:orc` or `--gc:arc`.
+
+Compatibility notes:
+- `--newruntime` and `--refchecks` are deprecated,
+  use `--gc:arc`, --gc:orc`, or `--gc:none` as appropriate instead.
+
+
+## Docgen
+- docgen: RST files can now use single backticks instead of double backticks and correctly render
+  in both `rst2html` (as before) as well as common tools rendering RST directly (e.g. GitHub), by
+  adding: `default-role:: code` directive inside the RST file, which is now handled by `rst2html`.
+
+- Source+Edit links now appear on top of every docgen'd page when
+  `nim doc --git.url:url ...` is given.
+
+- Latex doc generation is revised: output `.tex` files should be compiled
+  by `xelatex` (not by `pdflatex` as before). Now default Latex settings
+  provide support for Unicode and do better job for avoiding margin overflows.
+
+- The RST parser now supports footnotes, citations, admonitions, short style references with symbols.
+- The RST parser now supports Markdown table syntax.
+  Known limitations:
+  - cell alignment is not supported, i.e. alignment annotations in a delimiter
+    row (`:---`, `:--:`, `---:`) are ignored,
+  - every table row must start with `|`, e.g. `| cell 1 | cell 2 |`.
+
+- Implemented `doc2tex` compiler command which converts documentation in
+  `.nim` files to Latex.
+
+- docgen now supports syntax highlighting for inline code.
+- docgen now supports same line doc comments:
+```nim
+func fn*(a: int): int = 42  ## Doc comment
+```
+- docgen now renders deprecated and other pragmas.
+
+- `runnableExamples` now works with templates and nested templates.
+- `runnableExamples: "-r:off"` now works for examples that should compile but not run. 
+- `runnableExamples` now renders code verbatim, and produces correct code in all cases.
+- docgen now shows correct, canonical import paths in docs.
+- docgen now shows all routines in sidebar, and the proc signature is now shown in sidebar.
+
+## Effects and checks
+- Significant improvement to error messages involving effect mismatches
+
+- There is a new `cast` section `{.cast(uncheckedAssign).}: body` that disables some
+  compiler checks regarding `case objects`. This allows serialization libraries
+  to avoid ugly, non-portable solutions. See RFC [#407](https://github.com/nim-lang/RFCs/issues/407).
+  for more details.
+
+## Tools
+- major improvements to `nimgrep`, see PR [#15612
+](https://github.com/nim-lang/Nim/pull/15612).
+- `fusion` is now un-bundled from Nim, `./koch fusion` will
+  install it via Nimble at a fixed hash.
+
+- `testament`: added `nimoutFull: bool` spec to compare full output of compiler
+  instead of a subset; many bugfixes to testament.
+
+- Added `atlas` a tool for automating workflows involing "cloning" a nimble package and dependencies recursively
```

Comment:
```suggestion
- Added `atlas` a tool for automating workflows involving "cloning" a nimble package and dependencies recursively
```

Reply:
If atlas is for internal use then I would just avoid mentioning it

Reply:
based on the discussions from this PR https://github.com/nim-lang/Nim/pull/18756#issuecomment-908088929, I have to agree with you; I've now removed the mention